dbms/src/Functions/greatCircleDistance.cpp

```diff
@@ -24,7 +21,91 @@ namespace ErrorCodes
     extern const int LOGICAL_ERROR;
 }
 
-static inline Float64 degToRad(Float64 angle) { return angle * DEGREES_IN_RADIANS; }
+static const double PI = 3.14159265358979323846;
```

Comment:
Authorship and license must be specified.

(Andrey Aksenov, the author of original code, permitted to use this code in ClickHouse under the Apache 2.0 license.)

Reply:
The link to presentation from HighLoad++ Siberia an a video is also valuable.

Reply:
Also add a link to the original code.